rfcs/0044-node-status-backend.md

```diff
@@ -0,0 +1,36 @@
+## Summary
+
+[summary]: #summary
+
+This RFC proposes that we use AWS stack to be the backend of the node status/error system. To be specific by AWS stack, I mean we would use S3 as the storage backend; OpenSearch as the search engine; Kibana as the visualization and charting tool; we could also setup the miniservice that listening on the minaprotocol/node-status on AWS ES2.
+
+## Motivation
+
+[motivation]:#motivation
+
+We need a backend for node staus/error systems. Candidates for the backend include AWS stack and Grafana Loki and LogDNA.
+
+## Implementation
+
+[implementation]:#implementation
+
+The implementation is mainly to setup a miniservice under a domain like https://minaprotocol.com/node-status-service that would validate the report that sends by the users and maybe setup some kind of DOS protection (help would be need to design and implement in this category) and then sends those to the AWS Kenesis stream.
```

Comment:
I would recommend using a subdomain, like "node-status.minaprotocol.com"

Can you change the language to be more decisive here? I think the intent of the RFC is to understand what the plan is before implementation. So rather than saying "setup a miniservice under a domain like htttps://..." change to "a microservice will be deployed at https://node-status.minaprocotol.com ".

Same with "maybe setup some kind of DOS protection": This RFC should make a definite proposal one way or another. "We do not need to implement DOS protection because x" or "We are implementing DOS protection and here are details on how it will work"

Reply:
Also, can you include more detail around the rationale here? Why are we using the AWS stack over the other choices here?

Reasoning should speak to all of the requirements in the node-status / node-error product reqs so we can be sure that the implementation will fulfill our needs

Reply:
I think there's supposed to be a "Rationale and Alternatives" section that would be a good place for the reasoning